runner.py

```diff
@@ -13,11 +15,25 @@
 
     args = parser.parse_args()
     cookie_path, number_of_persons, folder_path, mode_flag, loop_flag, base_delay, variance = args.t, args.n, args.f, args.m, args.l, args.d, args.dp
-    running = True
-    while running:
-        base64_media = start_scrape(cookie_path, folder_path, number_of_persons, mode_flag)
-        if loop_flag == "single": running = False
-        elif loop_flag == "loop":
-            variance = int(base_delay / 100 * variance)
-            delay = base_delay + random.randint(-variance, variance)
-            time.sleep(delay)
+
+    if os.path.exists("token.txt"):
+        running = True
+        while running:
+            base64_media = start_scrape(cookie_path, folder_path, number_of_persons, mode_flag)
+            if loop_flag == "single": running = False
+            elif loop_flag == "loop":
+                variance = int(base_delay / 100 * variance)
+                delay = base_delay + random.randint(-variance, variance)
+                time.sleep(delay)
+    elif args.user and args.passwd != None:
```

Comment:
Instead of the if os.path.exists else just get the session_id if token.txt is missing at the start and then you can always use it